src/docs/product/alerts/create-alerts/metric-alert-config.mdx

```diff
@@ -59,6 +50,33 @@ Choose the time period over which to evaluate your metric. Your choices range be
 - At 3:02pm: 2:02pm - 3:02pm
 - ...
 
+## Filters
+
+The following set of filters translates into a [Discover query](/product/discover-queries/) that is displayed in the chart at the top of the alert configuration page.
+
+### Environment
+
+Specify which <PlatformLink to="/configuration/environments/">environment(s)</PlatformLink> will use this particular alert rule. This control filters on the `environment` tag in your events. This filter is helpful because the urgency and workflows you apply to production alerts might differ from those you apply to alerts originating from your QA environment, for example.
+
+The “Environment” dropdown list here has the same environments that are available for the selected project in the common “Environment” filter dropdown (this does not include hidden environments). Selecting "All Environments" is equivalent to having no environment filter.
+
+## Project
+
+Specify which project will use this particular alert rule. Created alert rule will only process events from this project.
+
```

Comment:
```suggestion

### Environment

Specify which <PlatformLink to="/configuration/environments/">environment(s)</PlatformLink> will use this particular alert rule. This control filters on the `environment` tag in your events. This filter is helpful because the urgency and workflows you apply to production alerts might differ from those you apply to alerts originating from your QA environment, for example.

The “Environment” dropdown list here has the same environments that are available for the selected project in the common “Environment” filter dropdown (this does not include hidden environments). Selecting "All Environments" is equivalent to having no environment filter.

```

Reply:
:+1: